### ISSUE TITLE
Reduce redundant code

### ISSUE THREAD

**Issue by KartikVerma0** [vasani-arpit/WBOT#375]
**Is your feature request related to a problem? Please describe.**
There are several pieces of code that are redundant and are not necessary like in utils file you have externalInjection function which is used in index.js to import appconfig which can be directly imported

**Describe the solution you'd like**
I would like to clear out redundant code 

**Describe alternatives you've considered**
like use require function to directly import json file rather than making custom externalInjection function

**Additional context**

**Comment by KartikVerma0** [comment]
I think reducing redundant code can make it simpler for developers to understand the code base, and I would like to work on this issue.

**Comment by vasani-arpit** [comment]
Yes. webhooks used to be called from chrome itself so that that time config was needed there, earlier version of puppeteer wasn't supporting direct code injection so had to write util for that. 

Feel free to make a PR for whatever you think is extra or unused.
